### ISSUE TITLE
Questions about the implementation details.

### ISSUE THREAD

**Issue by SunSwallow** [AuroraZengfh/RobustMerge#10]
Hi, thanks for releasing this implementation and the RobustMerge paper! While reading the code, I noticed two potential difference and wanted to confirm whether they are intentional design choices or implementation shortcuts:

**1. S_vector normalization**:
In the paper, the scaling coefficients are calculated (eq3) normalized acrossed tasks (eq4).  In the code, s_vector is computed similarly, but instead of cross-task normalization it is passed through a custom ``clamp`` function:

```python
s_vector = torch.sum(abs(concat_model_weight),dim=-1)/ torch.sum(abs(masks * concat_model_weight), dim=-1)
scale = clamp(s_vector, 1-att_ratio, 0)
```

The ``clamp`` uses per-row quantile clipping (e.g., min at 80% quantile when ``att_ratio=0.2``), without explicit cross-task normalization. Is this intended to approximate Eq.4, or is it a divergence from the paper?

**2. Merging in A/B parameter space vs. $\Delta W$ space**
The paper merges  $\Delta W_n$ while in the code, it seems that all ``lora_A`` are merged into a single $A^*$ (same for B), and the final adapter is a single LoRA rather than a sum of per-task $\Delta W$.

Could you clarify whether these differences from the method section are intentional, or there is any misunderstanding?

Thanks!

**Comment by AuroraZengfh** [comment]
Hi @SunSwallow. Thanks for your interest. Regarding the problems:
1. It is a trivial operation without any insight. You can try if removing it make any difference.
2. It seems that I do not get the exact point. I indeed merge the sum of $$\Delta W$_n$. Could you explain more?

**Comment by SunSwallow** [comment]
Sorry, but i didn't get the point. Even in the ``lora_B`` branch, 
```python
merged_model_weights[name]=torch.sum(fuse_weight * scale.unsqueeze(1) * trimed_model_weights, dim=0)/torch.sum(scale.unsqueeze(1), dim=0)
```
also calculates the sum of ``lora_B`` weights. And the final saved ``merged_model_weights`` only contains LoRA module keys. So the code only saves lora weights instead of ΔW?

**Comment by SunSwallow** [comment]
Thanks for your clarification!
